pkgs/applications/misc/home-manager/default.nix

```diff
@@ -0,0 +1,51 @@
+#Taken directly from
+#https://github.com/rycee/home-manager/blob/9c1b3735b402346533449efc741f191d6ef578dd/home-manager/default.nix
+
+{ pkgs
```

Comment:
The arguments should be made more "standard". I.e., instead of pulling everything from `pkgs`, should include `stdenv`, `fetchFromGitHub`, `bash`, `coreutils`, etc. here.